src/Controller/CheckoutRequestController.php

```diff
@@ -252,4 +254,24 @@ protected function processException($errorType, $errorMessage, \Exception $excep
 
         return $this->getResponse()->withJson($errorResp)->withStatus($statusCode);
     }
+
+    /**
+     * @param array $data should have a patronBarcode attribute, which will be randomly reassigned for partners
+     * @return array will be a copy of the input with patronBarcode reassigned in case it was originally a partner barcode
+     */
+    public function reassignPartnerBarcode($data)
+    {
+      $key = "PATRON_BARCODES_{$data['patronBarcode']}";
+      $barcodes = explode("," , Config::get($key, "", true));
```

Comment:
I think a better empty value for the second parameter probably should be `null`. It fits the definition of the function `get` in `NYPL\Starter\Config`

```
public static function get($name = '', $defaultValue = null, $isEncrypted = false)
```

Reply:
I def prefer `null` over an empty value, but in this case we're going to call `explode` on that default value. PHP's `explode` seems to interpret splitting `null` as being equivalent to splitting `''`, but that seems less clear to me (i.e. it surprises me that splitting `null` behaves this way). I think given that the [official doc on `explode`](http://php.net/explode) declares the param a `string`, we should probably send it a string so that we know what we're going to get back.